README.md

```diff
@@ -34,6 +34,7 @@ Ultimately, the information that needs to be in the charter is:
   - What actions are you proposing the WG be allowed to take directly?
   - Which actions will the WG take back to the Board for votes?
 - **Initial membership** - who will be in this working group when it's first created?
+- **Board of liaison** - the person will be the bridge between the DSF board and the working group. It will be someone who will keep the vision of the board in the working group and have same rights as a member of the working group.  
```

Comment:
```suggestion
- **Board of liaison** - the person will be the bridge between the DSF board and the working group. It will be someone who will keep the vision of the board in the working group and have the same rights as a member of the working group.  
```